src/test/cypress/integration/exam/ExamAssessment.spec.ts

```diff
@@ -0,0 +1,175 @@
+import { generateUUID } from '../../support/utils';
+import { artemis } from '../../support/ArtemisTesting';
+import { CypressExamBuilder } from '../../support/requests/CourseManagementRequests';
+import { BASE_API, GROUP_SYNCHRONIZATION, PUT } from '../../support/constants';
+import dayjs from 'dayjs';
+import { ProgrammingExerciseSubmission } from '../../support/pageobjects/OnlineEditorPage';
+import partiallySuccessful from '../../fixtures/programming_exercise_submissions/partially_successful/submission.json';
+import textSubmission from '../../fixtures/text_exercise_submission/text_exercise_submission.json';
+
+// requests
+const courseManagementRequests = artemis.requests.courseManagement;
+
+// page objects
+const examStartEnd = artemis.pageobjects.examStartEnd;
+const modelingEditor = artemis.pageobjects.modelingEditor;
+const modelingAssessment = artemis.pageobjects.modelingExerciseAssessmentEditor;
+const editorPage = artemis.pageobjects.onlineEditor;
+const assessmentDashboard = artemis.pageobjects.assessmentDashboard;
+
+// Common primitives
+const admin = artemis.users.getAdmin();
+const student = artemis.users.getStudentOne();
+const tutor = artemis.users.getTutor();
+let exam: any;
+let exerciseGroup: any;
+let course: any;
+let examTitle: string;
+
+describe('Exam Assessment', () => {
```

Comment:
Currently the text exerise assessment page throws an uncaught exception when making a request to athene, which results in cypress failing the test. This is a workaround and should be removed after the issue is fixed
```suggestion
let examTitle: string;

// This is a workaround for uncaught athene errors. When opening a text submission athene throws an uncaught exception, which fails the test
Cypress.on('uncaught:exception', () => {
    return false;
});

describe('Exam Assessment', () => {
```